rtengine/ipwavelet.cc

```diff
@@ -2265,11 +2263,7 @@ void ImProcFunctions::ip_wavelet(LabImage * lab, LabImage * dst, int kall, const
                                 absciss = asig * std::fabs(tempwav) + bsig;
                             } else {
                                 absciss = amean * std::fabs(tempwav);
-                                float k = sig;
-                                if(sig > 1.f) {
-                                    k = SQR(sig);
-                                }
-                                float abs = pow(2.f * absciss, (1.f / k));
+                                float abs = pow(2.f * absciss, (1.f / SQR(sig)));
```

Comment:
It looks as if the value of sig is always 2.0f... so what is going on anyway?

Reply:
good catch, looking

Reply:
@Thanatomanic Not sure, that my change (sqrt(sqrt(...))) is what you had in mind, but it's fast ;-)

Reply:
@Thanatomanic My intention was to get the files cppcheck clean. It we want them to get perfectly clean (not only cppcheck...) we will get from 100th to 1000th issue and will not be able to do this for 5.9 ;-)